### DISCUSSION TITLE
Any plan to fix "KafkaConsumer should not throw away already fetched data for paused partitions."

### DISCUSSION THREAD

**Discussion by jnorcross** [edenhill/librdkafka#4659] [Q&A]
This issue appears to have been fixed in the Java version here:
https://issues.apache.org/jira/browse/KAFKA-7548

Is there any plan to fix this in the C/C# version of the clients too?

**Comment by mensfeld** [comment]
I would love to see this one as well. Even if seeking would still purge, paused and resumed should not cause this imho.

**Comment by nicholasngai** [comment]
Hi folks! Also hoping to get some movement on this. We pause and resume frequently and would love to stop throwing away fetched data.

**Comment by nicholasngai** [comment]
I opened an issue against this discussion here BTW:

https://github.com/confluentinc/librdkafka/issues/5343
